tests/utils_tests/test_flatten.py

```diff
@@ -0,0 +1,29 @@
+from unittest import TestCase
+
+from django.utils.functional import flatten
+
+
+def _fl(x):
+    return list(flatten(x))
+
+
+class FlattenTestCase(TestCase):
+    def test_array(self):
+        self.assertEqual([1, 2], _fl([1, 2]))
+
+    def test_nested_array(self):
+        self.assertEqual([1, 2, 3, 4], _fl([[1, 2], [3, 4]]))
```

Comment:
Imho this needs several of more tests:
- test for empty containers
- test the seqtypes argument
- test for all types treated as seqtypes